core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -1223,6 +1223,14 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
     case "TRUE":
       unparseBoolean(writer, call);
       break;
+    case "GETBIT":
+      call.operand(0).unparse(writer, leftPrec, rightPrec);
```

Comment:
Kindly make a separate function for unparsing the GETBIT call

Reply:
Done